cmd/user-agent/src/store/modules/options.js

```diff
@@ -50,6 +51,31 @@ export default {
                     })
 
                 agentOpts['http-resolver-url'] = agentOpts['http-resolver-url'].split(',')
+
+                const userInfoURL = agentOpts["edge-agent-server"] + "/oidc/userinfo"
+
+                console.log("User info URL is: " + userInfoURL)
+
+                const client = axios.create({
+                    withCredentials: true
+                })
+
+                await client.get(userInfoURL)
+                    .then(resp => {
+                        const edvVaultURL = resp.data.bootstrap.edvVaultURL
```

Comment:
We're going to need alignment at some point w.r.t. the bootstrap "URLs" (values the agent cannot change at hub-auth) vs. the vault and keystore IDs.

I keep seeing usage of what are supposed to be just "vault/keystore IDs" being used to store the full URL.

cc @Baha-sk

Reply:
vaultID, keystoreID are supposed to be the full URL to the vault/keystore @llorllale 

The user should not be able to modify this value after onboarding. Only admins can update these values for migration purposes.

cc @DRK3 @aholovko @rolsonquadras